mobile/e2e-tests/scripts/redeploy_mentra_mini.sh

```
@@ -0,0 +1,160 @@
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(cd "$SCRIPT_DIR/../../.." && pwd)"
+
+HOST="mentra-mini"
+REMOTE_REPO='~/Documents/MentraOS'
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)